webapp/src/components/call_error_modal.tsx

```diff
@@ -125,6 +127,26 @@ export const CallErrorModal = (props: Props) => {
         modalProps.cancelButtonText = formatMessage({defaultMessage: 'Cancel'});
         confirmMsg = formatMessage({defaultMessage: 'Learn more'});
         break;
+    case userRemovedFromChannelErr.message:
+        headerMsg = (
+            <span>{formatMessage({defaultMessage: 'You were removed from channel'})}</span>
+        );
+        msg = (
+            <span>
+                {formatMessage({defaultMessage: 'You were disconnected from the call as you were removed from the channel.'})}
+            </span>
+        );
+        break;
+    case userLeftChannelErr.message:
+        headerMsg = (
+            <span>{formatMessage({defaultMessage: 'You left the channel'})}</span>
+        );
+        msg = (
+            <span>
+                {formatMessage({defaultMessage: 'You were disconnected from the call as you left the channel.'})}
+            </span>
+        );
+        break;
```

Comment:
@cwarnermm This is the part I am asking for editor review as we are adding a couple of new strings. Thanks!